entries/n/ngpvan.com.json

```
@@ -0,0 +1,10 @@
+{
+  "Site Name": {
+    "domain": "ngpvan.com",
+    "img": "ngpvan.com.png",
+    "tfa": ["sms", "totp"],
+    "url": "https://myaccount.ngpvan.com/Account",
+    "documentation": "https://www.ngpvan.com/actionid/",
+    "categories": ["hosting", "marketing", "other"]
+  }
```

Because ActionID is only relevant to the U.S., a `regions` array should be added.

```suggestion
    "regions": [
      "us"
    ]
  }
```

Just adding this suggestion will cause the JSON to break because of a missing comma after the `categories` element. Make sure to add the comma manually.